spec/requests/api/v1/user_request_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::UserController, type: :request do
+  describe 'GET /api/v1/user' do
+    let!(:user) { create(:user) }
+    let(:object) { JSON.parse(response.body)}
+
+    it 'Should return all users' do
+      get '/api/v1/user', params: {
+        email: user.email, encrypted_password: user.encrypted_password
```

Comment:
How can you send the encrypted password in params?  This don’t roller requires a user to be authenticated so you need to pass in credentials in headers.